utils/position_utils.tsx

```diff
@@ -6,6 +6,7 @@ export function popOverOverlayPosition(
     innerHeight: number,
     spaceRequiredAbove: number,
     spaceRequiredBelow?: number,
+    defaultHorizontalPosition?: 'left' | 'right',
```

Comment:
IMO this should be renamed to `horizontalPosition`, since a default should be done as a variable inside the component, or as a constant imported to it. I know the reason behind it, and it might be only my personal taste/style, but it does not feel right to name it that way.